scripts/finalizeRelease.ts

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env ts-node
+/* eslint-disable no-console */
+import * as readline from 'readline';
+import simpleGit from 'simple-git';
+import version from '../package.json';
+
+/**
+ * Script: upgradeConfigs.ts
+ * Run with: ts-node scripts/upgradeConfig.ts
+ * ------------------------------------------
+ * Amend the previous commit with changes to the changelog
+ * Then tag and push it. This script is run after
+ * `npx commit-and-tag-version`
+ */
+
+/**
+ * Amends the previous commit, creates the annotated release tag
+ * and then pushes commit and tag.
+ */
+async function commitAndTag(): Promise<void> {
+  await simpleGit().commit([], 'CHANGELOG.md', { '--amend': null, '--no-edit': null, '--no-verify': null });
+  await simpleGit().addAnnotatedTag(`v${version.version}`, `Release Version ${version.version}`);
```

Comment:
I'm confused by the `version.version`. Does this mean the first `version` is actually the contents of the entire package.json?

Reply:
Forgot I could also just `import { version } from '../package.json'`
Will fix.